src/lib/command-utils.ts

```diff
@@ -206,4 +206,10 @@ export function configName(): string {
   }
 
   return process.env.SWANKY_CONFIG?.split("/").pop() ?? DEFAULT_CONFIG_NAME;
+}
+
+export function swankyNodeCheck(config: SwankyConfig){
```

Comment:
It is time to reorg *task.ts* and *command-utils.ts*. We can create a *src/lib/config.ts* for config checkers. In that case you can rename it `ensureSwankyNodeInstalled` and move it to the *src/lib/config.ts* file.